src/include/migraphx/raw_data.hpp

```diff
@@ -104,6 +105,34 @@ struct raw_data : raw_data_base
         visit(v, [&](const auto&) { MIGRAPHX_THROW("Invalid tuple type"); });
     }
 
+    /**
+     * Visit the data with a tensor_view<uint8_t> with lens = {num bytes};
```

Comment:
Doesnt this convert it to `tensor_view<byte>`?

Reply:
Yeah, it's a `tensor_view<byte>` but the shape of the tensor view has a uint8_type. I'll update the comment